server/genome/hg38.ts

```diff
@@ -1,4 +1,4 @@
-const clinvar = require('../dataset/clinvar')
+import { clinsig }  from '../dataset/clinvar'
```

Comment:
@aacic Thanks very much for this! This is great that we are finally implementing syntax quality control. Hopefully we can add to the `utils/hooks/pre-commit`.

Reply:
I've tried to add following lines to .pre-commit-config.yaml
```
- repo: https://github.com/typescript-eslint/typescript-eslint
  rev: v5.59.11
  hooks:
    - id: typescript-eslint
```
But, I get this error:
```
An error has occurred: InvalidManifestError: 
=====> /Users/aacic/.cache/pre-commit/repoxxrx39jw/.pre-commit-hooks.yaml is not a file
Check the log at /Users/aacic/.cache/pre-commit/pre-commit.log
```
It seems that pre-commit doesn't work well with typescript-eslint. I would say we try to set-up Husky in another PR.